### PR TITLE
Add RUNTIME_TOOLS_LINK in Makefile

### PR DESCRIPTION
**Pull request by hqhq**
So we can build runtime_tools project anyway.

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,6 +25,10 @@ install: man
 
 clean:
 	rm -f oci-runtime-tool runtimetest *.1
+	rm -f $(RUNTIME_TOOLS_LINK)
+
+$(RUNTIME_TOOLS_LINK):
+	ln -sfn $(CURDIR) $(RUNTIME_TOOLS_LINK)
```
Comment (wking) [inline Makefile]: `-n` is [not in POSIX][1].  Do we really need it to support this use-case?

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/ln.html

**Comment by wking** [comment]
This feels sort of like opencontainers/image-tools#52.  runtime-tools
builds fine as it stands if you've checked it out to an appropriate
location in your $GOPATH.

**Comment by hqhq** [comment]
@wking It's ok we don't use `-n`, I've updated. And I know runtime-tools builds fine if I checked it out to an appropriate location in my $GOPATH, the point is users don't have checked it out to an appropriate location in their $GOPATH and they can still build runtime-tools.

It makes it more convenient like what we did in runc.

**Comment by mrunalp** [comment]
LGTM

**Comment by caniszczyk** [comment]
LGTM

**Comment by vbatts** [comment]
LGTM

**Comment by vbatts** [comment]
@caniszczyk FYI @mrunalp's LGTM did not trigger pullapprove

**Comment by wking** [comment]
> @mrunalp's LGTM did not trigger pullapprove

His LGTM predates #261 landing, so he wasn't cleared vs PullApprove when he made the comment.

**Pull request merged**